directory-connector/src/com/untangle/app/directory_connector/ActiveDirectoryManagerImpl.java

```diff
@@ -378,15 +382,18 @@ public JSONArray getUsers( String domain)
         throws ServiceUnavailableException
     {
         JSONArray result = new JSONArray();
-
+    
         for(ActiveDirectoryLdapAdapter adAdapter : this.adAdapters){
             if(adAdapter == null){
                 continue;
             }
             if(!adAdapter.getSettings().getEnabled()){
                 continue;
             }
-
+            
+            if(domain == "" || adAdapter.getSettings().getLDAPHost() == "" || adAdapter.getSettings().getLDAPHost() == null){
```

Comment:
Use StringUtils.EMPTY constant for empty string "".
Use StringUtils.isEmpty(adAdapter.getSettings().getLDAPHost()) for null and empty check.